pkgs/applications/version-management/gitkraken/default.nix

```diff
@@ -65,8 +65,8 @@ stdenv.mkDerivation rec {
     comment = "Graphical Git client from Axosoft";
   };
 
-  nativeBuildInputs = [ makeWrapper ];
-  buildInputs = [ dpkg ];
+  nativeBuildInputs = [ makeWrapper wrapGAppsHook ];
+  buildInputs = [ dpkg gtk3 gnome3.defaultIconTheme hicolor-icon-theme ];
```

Comment:
Please use `gnome3.adwaita-icon-theme` as `defaultIconTheme` is a deprecated alias.